addon/components/r-avatar/component.js

```diff
@@ -21,24 +21,17 @@ export default Component.extend({
 
   init() {
     this._super(...arguments);
-    this.attributes = {
+    this.allowedAttributes = {
       size: ['small', 'medium', 'large', 'x-large']
     };
     this.defaults = { size: 'medium' };
 
-    // sets appropriate properties to true for classNameBindings
-    const attributes = get(this, 'attributes');
+    const allowedAttributes = get(this, 'allowedAttributes');
     const defaults = get(this, 'defaults');
+    const suppliedAttrs = get(this, 'attrs');
```

Comment:
What is attrs? Are you proposing changing the component interface? e.g
{{r-button 
  attrs={
    size: 'small',
    variant: 'secondary'
}}